### DISCUSSION TITLE
TS构建问题

### DISCUSSION THREAD

**Discussion by janryWang** [alibaba/formily#1450] [Ideas]
原本写法
![image](https://user-images.githubusercontent.com/4060976/118689315-6e69b480-b839-11eb-93b7-5018ddebd8a7.png)

构建产物
![image](https://user-images.githubusercontent.com/4060976/118689253-5f830200-b839-11eb-90aa-bf1712a6f299.png)

这样明显不对的，应该是编译成@formily/core/esm/types这样依赖方式，但是目前我也没想到啥好办法

**Comment by liuweiGL** [comment]
是不是因为使用的是 namespace 导致的啊，使用 inport 把 GeneralField 导进来使用呢？

**Comment by liuweiGL** [comment]
解决方式是什么啊，看提交记录改的 tsconfig.json 配置？

**Reply by janryWang** [reply]
对

**Comment by liuweiGL** [comment]
https://github.com/alibaba/formily/commit/47eb9cd5af5c500701c66824422eb30f52eee3b3
